packages/kit/src/runtime/client/client.js

```diff
@@ -75,6 +74,9 @@ export function create_client({ target, base }) {
 	/** @type {Array<((url: URL) => boolean)>} */
 	const invalidated = [];
 
+	/** @type {import('svelte').SvelteComponent[]} */
+	const components = [];
```

Comment:
I think this deserves a comment how this is updated. I was super confused for a few minutes why this works, until I saw that it's manipulated within `Root.svelte`.